lib/internal/test_runner/runner.js

```diff
@@ -87,14 +87,24 @@ function createTestFileList() {
     cwd,
     exclude: (name) => name === 'node_modules',
   });
-  const results = glob.globSync();
 
-  if (hasUserSuppliedPattern && results.length === 0 && ArrayPrototypeEvery(glob.matchers, (m) => !m.hasMagic())) {
-    console.error(`Could not find '${ArrayPrototypeJoin(patterns, ', ')}'`);
-    process.exit(kGenericUserError);
+  if (hasUserSuppliedPattern && ArrayPrototypeEvery(glob.matchers, (m) => !m.hasMagic())) {
+    return Readable.from((async function*() {
```

Comment:
```suggestion
    return Readable.from((async * () => {
```

Reply:
why is this resolved? this function isn't recursive nor does it use `this` nor is `new`ed, so why does it need to be a non-arrow function?

Reply:
so apparently you can't do generator arrow function let alone `async` ones

Reply:
ohhhh duh, i knew that

Reply:
was resolved cause I clicked the commit suggestion and it marked as resolve automatically but then it failed to understand so changed back